### PR TITLE
feat(perf/go-libp2p): add v0.28.1

### PR DESCRIPTION
**Pull request by mxinden**
Adds the latest [go-libp2p release](https://github.com/libp2p/go-libp2p/releases/tag/v0.28.1) to the performance benchmarking setup.

### CODE REVIEW

**Comment by mxinden** [comment]
@libp2p/go-libp2p-maintainers can one of you give this a review?

File: perf/impl/go-libp2p/v0.27/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/libp2p/test-plans/perf/go/v0.27/v2
+module github.com/libp2p/test-plans/perf/impl/go-libp2p/v0.27
```
Comment (mxinden) [inline perf/impl/go-libp2p/v0.27/go.mod:1]: here

File: perf/impl/go-libp2p/v0.28/go.mod [hunk 2]
```diff
@@ -0,0 +1,94 @@
+module github.com/libp2p/test-plans/perf/impl/go-libp2p/v0.28
```
Comment (mxinden) [inline perf/impl/go-libp2p/v0.28/go.mod:1]: and here

**Review by mxinden — COMMENTED** [review]
As far as I can tell this updated module name is more Golang idiomatic. Happy for input.

File: perf/impl/go-libp2p/v0.27/go.mod (hunk 1 above)
Comment (marten-seemann) [reply perf/impl/go-libp2p/v0.27/go.mod:1]: That's correct.

Comment (marten-seemann) [reply perf/impl/go-libp2p/v0.27/go.mod:1]: That's correct.

**Review by marten-seemann — APPROVED** [review]

**Pull request merged**